policy/modules/services/pacemaker.te

```diff
@@ -24,20 +33,28 @@ files_tmpfs_file(pacemaker_tmpfs_t)
 type pacemaker_var_lib_t;
 files_type(pacemaker_var_lib_t)
 
+type pacemaker_var_log_t;
```

Comment:
Please remove the `var` from the type name. I don't want more type names that encode a path like this.

Reply:
While I'm making changes to these modules, do you want a patch renaming corosync_var_log_t to corosync_log_t?

Reply:
No. I'd rather do renames like that separately, and across the entire policy.